test/e2e/reboot.go

```diff
@@ -41,6 +41,9 @@ const (
 
 	// How long pods have to be "ready" after the reboot.
 	rebootPodReadyAgainTimeout = 5 * time.Minute
+
+	// How long pods have to be "ready" initially before a reboot.
+	rebootPodReadyInitialTimeout = 2 * time.Minute
```

Comment:
1. Done
2. Because if we make it long, we won't (hopefully) have to tune this again.  As with other timeouts of this style, since we are polling, most of the time we won't burn that much time waiting for the condition to become true.